spec/negotiation.html

```diff
@@ -325,5 +325,35 @@ <h1>GetNumberOption ( _options_, _property_, _minimum_, _maximum_, _fallback_ )<
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-partitionpattern" aoid="PartitionPattern">
+      <h1>PartitionPattern ( _pattern_ )</h1>
+
+      <p>
+        The PartitionPattern abstract operation is called with argument _pattern_. The following steps are taken:
```

Comment:
I could add some more details about the return value if it would be helpful.

Reply:
This is a good opportunity to say something like,

> This abstract operation parses an abstract pattern string into a list of Records with two fields, `[[Type]]` and `[[Value]]`.  The syntax of the abstract pattern strings is an implementation detail and is not exposed to users of ECMA-402.